js/client-chat.js

```diff
@@ -420,6 +420,58 @@
 				challenge(targets);
 				return false;
 
+			case 'accept':
+				var userid = toId(target);
+				if (userid) {
+					var $challenge = $('.pm-window').filter('div[data-userid="' + userid + '"]').find('button[name="acceptChallenge"]');
+					if (!$challenge.length) {
+						app.addPopupMessage('You do not have any pending challenge from ' + target + ' to accept.');
+						return false;
+					}
+					$challenge[0].click();
+					return false;
+				}
+
+				var $challenges = $('.challenge').find('button[name=acceptChallenge]');
+				if (!$challenges.length) {
+					app.addPopupMessage('You do not have any pending challenges to accept.');
```

Comment:
Ditto your other popup messages.

Popup messages in general are really bad UI; they should in general be replaced with other things.